linera-service/src/storage.rs

```diff
@@ -165,11 +165,11 @@ impl FromStr for StorageConfig {
         }
         print!("available storage: memory");
```

Comment:
What is this? Did we mean to call `tracing::error`?

Reply:
Ok, let me do that. Sorry for the divergence with the standard.